jaeger-core/src/main/java/com/uber/jaeger/senders/HttpSender.java

```diff
@@ -44,7 +46,7 @@
   private static final TProtocolFactory PROTOCOL_FACTORY = new Factory();
   private static final TSerializer SERIALIZER = new TSerializer(PROTOCOL_FACTORY);
 
-  private final HttpClient httpClient = new DefaultHttpClient();
+  private final HttpClient httpClient = new DefaultHttpClient(new PoolingClientConnectionManager());
```

Comment:
Our dependency on `httpclient` seems to be a mess. We are depending transitively on the `httpclient` bought in by `libthrift`, which is bad.

The right way™ to fix this would be to depend on `httpclient` directly. Because `httpclient` is very popular, we should shade `httpclient` and depend on that instead, so that our clients don't run into dependency issues. 

I'll tackle this tomorrow. @yurishkuro any objections?